.travis.yml

```diff
@@ -57,6 +57,18 @@ script:
 - ./test.sh
 - if [ "$TRAVIS_OS_NAME" = "linux" ]; then PATH=$PATH:$(pwd) ./test_e2e.sh; fi
 
+jobs:
+  include:
+    - stage: Go Coverage
+      before_script:
+        - go get github.com/mattn/goveralls
+      script:
+        - make test-coverage
+        # todo(camilamacedo86): Add env var with repotoken value in trevis. It should not be a public info. Then, replace as follows
+        # todo(camilamacedo86): ReGen the token in https://coveralls.io/github/kubernetes-sigs/kubebuilder
+        # - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage-all.out -repotoken=$COVERALLS_TOKEN
+        - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage-all.out -repotoken=xaBu9eWjZIkVgBBAe5uLt2Y5IsATjrALo
```

Comment:
@alexeldeib could you give me a hand here for we move forward with?

Reply:
@camilamacedo86 the e2e are working now in https://github.com/kubernetes-sigs/kubebuilder/pull/1201 (pending `lgtm` for merge after I cleaned up commits)

is there something specific here I can help with? I'm not following

Reply:
HI @alexeldeib, 

I do not have access to env vars of Travis. So, I am not enable to create an env var as COVERALLS_TOKEN to store the token. Could you help with? See the above lines.

Also, would be great to get a help with the review/approval to merge it as well. 

Really tks.

Reply:
Unfortunately I don't have permissions on Travis either 🙁 I asked in #sig-testing if there is a way to add a secret value (although I guess we aren't really treating it as secret) to Prow. 

For now, you could add it on L16 as a plain text value?

Reply:
here: https://github.com/kubernetes-sigs/kubebuilder/pull/1207/files#diff-354f30a63fb0907d4ad57269548329e3R16

Reply:
Oh, I see your comment above to regenerate the token. I think we would need either @mengqiy or @DirectXMan12 to add it in Travis, but based on the latest news about Travis I want to see if we can get it in Prow quickly too.

Reply:
For Prow, the secret can be created manually by test-infra on-call if you ping them in slack. Let me know if you want help setting up a job to run coverage?

Reply:
Hi @alexeldeib, 

Could you share this news?

Reply:
basically concerns about this:
https://kubernetes.slack.com/archives/C09QZ4DQB/p1574447538420300

Reply:
seems like maybe it was a false alarm though?

Reply:
@alexeldeib, 
I am not sure if I follow. Note that, Trevis is free for OpenSource projects and could not find any official news saying that they will change that. I thought that you folks have another motivation to move to use prow. The subscription is just for private repos.

Reply:
I think we're fine, you can probably ignore that. And yes, the original motivation was separate.

Reply:
@alexeldeib has helped us migrated some tests to prow already.
I guess our plan is to have our "unit" test and e2e test on prow, so people can easier `/retest` the failing tests.
We will keep our macOS test on travis, since prow doesn't support it.
Regarding go coverage test, we will leave it wherever is more easy to setup.